### ISSUE TITLE
Move virtual machine guids to git ignored config file

### ISSUE THREAD

**Issue by JoshuaWierenga** [JoshuaWierenga/EfiSharp#10]
Currently, I have to include guids specific to setup in EfiSharp.csproj and either keep modified files around forever or commit them. It would be ideal if these could be read in at build time with an external config file that was on the .gitignore list.

**Comment by JoshuaWierenga** [comment]
It would probably be better to store both guids in one file and put comments with them. I tried using generated resource files but that requires that file to be built and it is full of unimplemented code. I also tried just making a single txt file and using File.ReadLines but so far it complains about System.IO.File despite it working in this method.
![image](https://user-images.githubusercontent.com/13083645/107136564-6629e100-68fc-11eb-9bee-0290cdd60fda.png)
